### ISSUE TITLE
[BUG] SfTabs - Jump effect on page reload

### ISSUE THREAD

**Issue by isntlazy** [vuestorefront/storefront-ui#1688]
**We can see a jump effect when reloading the page**
The problem is actually happens when `SfTabs` and all children `SfTab` components is mounted, but there is no active Tab on this point, only after mount `SfTabs` will set activeTab to one of his children tabs. Because of this we have jumping effect when loading the page.

**How to reproduce**
1. Just use `SfTabs` component with few `SfTab`s inside
2. Reload the Page

**Expected behavior**
The content and tabs titles should be shown at the same time

**Actual behavior**
First we see the Tabs titles without content and just after we can see the tabs with content (jumping effect)

**Code examples**
```
<SfTabs
  :openTab="activeTab"
  tabShowText="show"
>
  <SfTab title="Tab1 Title">
    <ComponentInTab1
    />
  </SfTab>
  <SfTab title="Tab2 Title">
    <ComponentInTab2
    />
  </SfTab>
</SfTabs>
```

**Screenshots**
![image](https://user-images.githubusercontent.com/23207978/107504693-5e8c6580-6ba4-11eb-8581-aa62492f834c.png)
![image](https://user-images.githubusercontent.com/23207978/107504726-6815cd80-6ba4-11eb-872d-68af3c73b4db.png)

**Devices (please complete the following information):**
 - Device: [Desktop]
 - OS: [Mac OS 10.15.7]
 - Browser: [chrome][88]

**Comment by AdamPawlinski** [comment]
I sdded another PR from branch which comes from hotfix and not develop

**Comment by Baroshem** [comment]
@AdamPawlinski 

If this bug fix was merged, can this issue be closed?

**Comment by AdamPawlinski** [comment]
Yes of course.
